aptos-move/block-executor/src/executor.rs

```diff
@@ -527,172 +532,136 @@ where
         block: &TP,
         num_workers: usize,
     ) -> Result<(), PanicOr<ParallelBlockExecutionError>> {
-        let mut block_limit_processor = shared_commit_state.acquire();
+        let block_limit_processor = &mut shared_commit_state.acquire();
+        let mut commit_side_effects = false;
```

Comment:
the name is confusing to me, it reads like commit is a verb, but I think it means side_effects_at_commit_time?